test/DebuggerCommon/async.js.dbg.baseline

```diff
@@ -0,0 +1,109 @@
+[
+  {
+    "callStack": [
+      {
+        "line": 10,
+        "column": 4,
+        "sourceText": "return await af1(10)",
+        "function": "af2"
+      },
+      {

```

Comment:
In PDM based debugger baselines I see a frame here "Generator.prototype.next" why is it not here?